### ISSUE TITLE
Weather MQTT endpoint does not match documentation

### ISSUE THREAD

**Issue by mich181189** [emfcamp/developer.emfcamp.org#54]
A few things:
* it looks like it sends JSON on "emf/weather" where the documentation says there are multiple topics
* It looks awfully like the temperature data in farenheit?

**Comment by andypiper** [comment]
Agreed with the observation that it is a single JSON message on that topic, not subtopics as documented. However, I'm seeing temperature values that look legitimately to be in Celsius (14C and 19C are believable current values from the data below)

In addition, there are a large number of undocumented values, and some documented-but-missing values (`eventrain`,`hourlyrain`,`dailyrain`,`weeklyrain` and 
`solarradiation_lux` are listed in the current docs, but not found in the output); many other values (`*piezo`, `safe_exposure*` and others) are not listed in the docs.

```json
{
  "runtime": 7338.0,
  "heap": 26000.0,
  "tempin": 19.699999999999996,
  "humidityin": 46.0,
  "baromrel": 1002.303309768129,
  "baromabs": 1002.303309768129,
  "temp": 14.200000000000001,
  "humidity": 59.0,
  "winddir": 203.0,
  "windspeed": 14.75768448,
  "windgust": 24.12406656,
  "maxdailygust": 24.12406656,
  "solarradiation": 178.45,
  "uv": 1.0,
  "rrain_piezo": 0.0,
  "erain_piezo": 0.0,
  "hrain_piezo": 0.0,
  "drain_piezo": 0.40640000000000004,
  "wrain_piezo": 11.5062,
  "mrain_piezo": 12.2936,
  "yrain_piezo": 12.2936,
  "ws90cap_volt": 5.3,
  "ws90_ver": 138.0,
  "wh90batt": 3.26,
  "interval": 20.0,
  "beaufortscale": 3,
  "dewpoint": 6.308491223066804,
  "feelslike": 14.200000000000001,
  "frostpoint": 4.471798694524923,
  "frostrisk": "No risk",
  "heatindex": 13.216111111111115,
  "humidex": 14,
  "humidex_perception": "Comfortable",
  "humidityabs": 7.201474941565666,
  "humidityabsin": 7.201474941565666,
  "relative_strain_index": null,
  "relative_strain_index_perception": null,
  "safe_exposure_time_skin_type_1": 166.7,
  "safe_exposure_time_skin_type_2": 200.0,
  "safe_exposure_time_skin_type_3": 266.7,
  "safe_exposure_time_skin_type_4": 333.3,
  "safe_exposure_time_skin_type_5": 533.3,
  "safe_exposure_time_skin_type_6": 866.7,
  "simmerindex": null,
  "simmerzone": null,
  "solarradiation_perceived": 87.07778922002475,
  "thermalperception": "Dry",
  "windchill": null
}
```

**Comment by chaos-dotcom** [comment]
Looks like the temp issue has been fixed - here is a grab from an earlier time, 



`{"runtime": 56436.0, "heap": 25836.0, "tempin": 68.18, "humidityin": 51.0, "baromrel": 29.589, "baromabs": 29.589, "temp": 59.54, "humidity": 60.0, "winddir": 275.0, "windspeed": 6.93, "windgust": 10.74, "maxdailygust": 20.13, "solarradiation": 257.14, "uv": 2.0, "rrain_piezo": 0.0, "erain_piezo": 0.008, "hrain_piezo": 0.004, "drain_piezo": 0.016, "wrain_piezo": 0.453, "mrain_piezo": 0.484, "yrain_piezo": 0.484, "ws90cap_volt": 5.3, "ws90_ver": 138.0, "wh90batt": 2.26, "interval": 20.0, "beaufortscale": 2, "dewpoint": 45.65598325602913, "feelslike": 59.54, "frostpoint": 42.069166658341956, "frostrisk": "No risk", "heatindex": 58.013999999999996, "humidex": 16, "humidex_perception": "Comfortable", "humidityabs": 0.0004889456451752023, "humidityabsin": 0.0004889456451752023, "relative_strain_index": null, "relative_strain_index_perception": null, "safe_exposure_time_skin_type_1": 83.3, "safe_exposure_time_skin_type_2": 100.0, "safe_exposure_time_skin_type_3": 133.3, "safe_exposure_time_skin_type_4": 166.7, "safe_exposure_time_skin_type_5": 266.7, "safe_exposure_time_skin_type_6": 433.3, "simmerindex": null, "simmerzone": null, "solarradiation_perceived": 90.25085096555111, "thermalperception": "Dry", "windchill": null}`


a grab at the current time of writing looks like 

`{"runtime": 8094.0, "heap": 25628.0, "tempin": 19.299999999999997, "humidityin": 48.0, "baromrel": 1002.303309768129, "baromabs": 1002.303309768129, "temp": 14.4, "humidity": 60.0, "winddir": 203.0, "windspeed": 14.75768448, "windgust": 24.12406656, "maxdailygust": 24.12406656, "solarradiation": 235.6, "uv": 2.0, "rrain_piezo": 0.0, "erain_piezo": 0.0, "hrain_piezo": 0.0, "drain_piezo": 0.40640000000000004, "wrain_piezo": 11.5062, "mrain_piezo": 12.2936, "yrain_piezo": 12.2936, "ws90cap_volt": 5.3, "ws90_ver": 138.0, "wh90batt": 3.26, "interval": 20.0, "beaufortscale": 3, "dewpoint": 6.740360413928867, "feelslike": 14.4, "frostpoint": 4.875453190578243, "frostrisk": "No risk", "heatindex": 13.462222222222222, "humidex": 14, "humidex_perception": "Comfortable", "humidityabs": 7.413814731987664, "humidityabsin": 7.413814731987664, "relative_strain_index": null, "relative_strain_index_perception": null, "safe_exposure_time_skin_type_1": 83.3, "safe_exposure_time_skin_type_2": 100.0, "safe_exposure_time_skin_type_3": 133.3, "safe_exposure_time_skin_type_4": 166.7, "safe_exposure_time_skin_type_5": 266.7, "safe_exposure_time_skin_type_6": 433.3, "simmerindex": null, "simmerzone": null, "solarradiation_perceived": 89.49096389649245, "thermalperception": "Dry", "windchill": null}`

**Comment by andypiper** [comment]
Wild (!?) guess / quick research suggests that is that this is an Ecowitt system using ecowitt2mqtt so [this documentation](https://github.com/bachya/ecowitt2mqtt?tab=readme-ov-file#calculated-sensors) looks reasonable for defining what the values should be.

**Comment by edwinbalani** [comment]
I think we're likely to use the same hardware in 2026, at a minimum - so worth a look at this to try getting it right this time.  I'm told the station we're using this year might support MQTT without a daemon in the middle to translate, so the output could be different but better maybe.
